marklogic-client-api/src/main/java/com/marklogic/client/impl/RESTServices.java

```diff
@@ -440,6 +440,20 @@ void patchDocument(RequestLogger reqlog, DocumentDescriptor desc, Transaction tr
     public String getParamName() {
       return paramName;
     }
+    public CallField toBuffered() {
+    	return this;
+    }
+ 
+    public interface MultipleAtomicCallField {
```

Comment:
Given that CallField is an abstract class, the story was wrong to suggest that MultipleAtomicCallField, MultipleNodeCallField, and SingleNodeCallField should be interfaces.

Instead, it would be better if these interfaces were static public abstract classes at the same level as CallField that extend CallField to create a proper type hierarchy.

That way, the Buffered and Unbuffered concrete classes can extend the specific abstract class instead of extending CallField but implementing the interface.